functions/go/inflate-helm-chart/README.md

```diff
@@ -0,0 +1,150 @@
+
+The `inflate-helm-chart` function inflates a local or remote Helm chart. 
+
+<!--mdtogo-->
+
+You can learn more about helm charts [here][helm].
+
+## Usage
+
+This function can be used with any KRM function orchestrators (e.g. kpt or kustomize) 
+to inflate a specified Helm chart.
+It can be used imperatively or declaratively when run with kustomize. With kpt, it can be
+run imperatively or declaratively as an exec function, or just imperatively as a container
+function. 
+
+### FunctionConfig
+
+<!--mdtogo:Long-->
+
+There are 2 kinds of `functionConfig` supported by this function:
+
+- `ConfigMap`
+- A custom resource of kind `HelmChartInflator`
+
+To use a `ConfigMap` as the `functionConfig`, the desired parameters must be
+specified in the `data` field:
+
+```yaml
+data:
+  chartHome: string
+  configHome: string
+  name: string
+  version: string
+  repo: string
+  releaseName: string
+  namespace: string
+  valuesInline: map[string]interface{}
+  valuesFile: string
+  valuesMerge: string
+  includeCRDs: bool
+```
+
+A `functionConfig` of kind `HelmChartInflator` has the following supported parameters: 
+
+```yaml
+helmGlobals:
+  chartHome: string
+  configHome: string
+helmCharts:
+- name: string
+  version: string
+  repo: string
+  releaseName: string
+  namespace: string
+  valuesInline: map[string]interface{}
+  valuesFile: string
+  valuesMerge: string
+  includeCRDs: bool
+```
+
+#### Parameter Descriptions
+
+| Field        |  Description | 
+| -----------: |  ----------- | 
+`helmGlobals`  | Parameters applied to all Helm charts
+`helmCharts`   |  An array of helm chart parameters
+`chartHome`    | A filepath to a directory of charts. The function will look for the chart in this local directory before attempting to pull the chart from a specified repo. Defaults to "tmp/charts". When run in a container, this path MUST have the prefix "tmp/".
+`configHome`   | Defines a value that the function should pass to helm via the HELM_CONFIG_HOME environment variable. If omitted, {tmpDir}/helm is used, where {tmpDir} is some temporary directory created by the function for the benefit of helm.
+`name`         | The name of the chart
+`version`      | The version of the chart
+`repo`         | A URL locating the chart on the internet
+`releaseName`  | Replaces RELEASE_NAME in the chart template output
+`namespace`    | Sets the target namespace for a release (.Release.Namespace in the template)
+`valuesInline` | Values to use instead of default values that accompany the chart
+`valuesFile`   | ValuesFile is local file path to a values file to use _instead of_ the default values that accompanied the chart. The default values are in '{ChartHome}/{Name}/values.yaml'.
+`valuesMerge`  | ValuesMerge specifies how to treat ValuesInline with respect to Values. Legal values: 'merge', 'override' (default), 'replace'.
+`includeCRDs`  | Specifies if Helm should also generate CustomResourceDefinitions, defaults to 'false'
+
+#### Example with kustomize
```

Comment:
Please follow our usage doc template (https://raw.githubusercontent.com/GoogleContainerTools/kpt-functions-catalog/master/functions/go/_template/README.md):

```
## Examples

<!--mdtogo:Examples-->
Your content here...
<!--mdtogo-->
```

There are 2 options (they are not mutually exclusive) for the example(s) using kustomize:
- Add example(s) in the `examples` directory. We can potentially test it in the CI.
- Add inline example(s) in the usage doc.

If we do the later, I'd suggest to add an small inline example for kpt as well.
@natasha41575 WDYT?

Reply:
I added inline examples, two for kustomize and one for kpt. The other kpt examples are still under `/examples`, LMK what you think